src/platform-implementation-js/dom-driver/gmail/widgets/buttons/button-view.ts

```diff
@@ -435,17 +435,22 @@ export default class ButtonView implements ButtonViewI {
         BUTTON_COLOR_CLASSES[this._buttonColor].HOVER_CLASS,
       );
       this._element.classList.remove('inboxsdk__button_hover');
+      this._element.classList.add(
+        BUTTON_COLOR_CLASSES[this._buttonColor].INACTIVE_CLASS,
+      );
     });
 
     Kefir.fromEvents(this._element, 'focus').onValue(() => {
-      this._element.classList.add('T-I-JO');
+      if (!this._buttonColor.match(/outline/)) {
+        this._element.classList.add('T-I-JO');
+      }
     });
 
-    Kefir.fromEvents(this._element, 'mouseup').onValue(() => {
-      if (!this._hasDropdown) {
+    if (!this._hasDropdown && !this._buttonColor.match(/outline/)) {
+      Kefir.fromEvents(this._element, 'mouseup').onValue(() => {
```

Comment:
Not sure if this mouse up handler is triggering this, but on keyboard nav, I'm seeing:

![2024-02-15 15 39 20](https://github.com/InboxSDK/InboxSDK/assets/5156873/da99c85d-46d6-41f5-ae68-01dbecf5719f)